app/models/merge_request.rb

```diff
@@ -10,15 +10,15 @@ class MergeRequest < ActiveRecord::Base
   # update the author name only once.
   attr_readonly :author_name
 
-  after_save :scan_description_for_issue_ids
+  after_save :scan_description_for_issue_ids, :update_mentioned_issues_status
 
   def self.find_all_by_issue(issue)
     includes(:issues).where(issues: { id: issue.id })
   end
 
   private
 
-  ISSUE_ID_REGEXP = /(?:[^a-z]|\A)(?:#|REDMINE-)(\d+)/
+  ISSUE_ID_REGEXP = /(?:#|REDMINE-)(\d+)/
```

Comment:
The removed part of the matcher excluded references to GitHub issues in other repositories of the form `org/project#123`. There is a [test which covers this case](https://github.com/tf/redmine_merge_request_links/blob/c101e6ae2894d977307702ae86167bf0467281d6/test/unit/merge_request_test.rb#L46) that fails now.

Reply:
Fixed.